lib/terraforming/resource/iam_user_policy.rb

```diff
@@ -16,22 +16,22 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_user_policy")
+        apply_template(@client, 'tf/iam_user_policy')
       end
 
       def tfstate
         iam_user_policies.inject({}) do |resources, policy|
           attributes = {
-            "id" => iam_user_policy_id_of(policy),
-            "name" => policy.policy_name,
-            "policy" => prettify_policy(policy.policy_document, breakline: true, unescape: true),
-            "user" => policy.user_name,
+            'id' => iam_user_policy_id_of(policy),
+            'name' => policy.policy_name,
+            'policy' => prettify_policy(policy.policy_document, breakline: true, unescape: true),
+            'user' => policy.user_name,
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)